### ISSUE TITLE
multiple proxypaths

### ISSUE THREAD

**Issue by MikeDabrowski** [txchen/light-server#32]
Hi, is there a way to set multiple proxy paths (array) ?
Something like:
```javascript
{
"port": 80,
 "serve": ".",
 "proxy": "http://localhost:8080",
 "proxypath": [
    "/api",
    "/user/login-bar.json",
    "/user/logout",
    "/ext/cms/payments/",
    "/authorize"
 ],
}
```

**Comment by txchen** [comment]
@MikeDabrowski The current proxy logic is: if static serve cannot find the resource, instead of returning 404, it will fallback to proxy url. 

So for your example, you don't need to specify the path list and it should still work. 

I think anyway you will not have static resources and API path share the same path. So, is this still a problem?

**Comment by txchen** [comment]
@MikeDabrowski sorry, I almost forgot I added --proxypath myself, because of the SPA mode, LOL.

So I guess this is a valid use case, I will think about implementing it.

**Comment by MikeDabrowski** [comment]
> So, is this still a problem?

It is not the problem at the moment. I needed that to test my app locally with all its features. I used xampp in the end. 
I think it would be nice to have that option.

**Comment by txchen** [comment]
Implemented in 2.3.0.
